.codespell-ignore-lines

```diff
@@ -15,4 +15,162 @@ Linix 45ZWN24-40           2  0.5 Ohm    0.400 mH  2.34A   24V
 * [#14540](https://github.com/apache/nuttx/pull/14540) CMake/preprocess: fix typo PREPROCES -> PREPROCESS
 * [#14927](https://github.com/apache/nuttx/pull/14927) spelling: fix spelling typo premption -> preemption
 * [#15520](https://github.com/apache/nuttx/pull/15520) drivers/note: fix typo falgs and align local name to irq_mask
+* [#4526](https://github.com/apache/nuttx/pull/4526) Rearch video
+* [#6447](https://github.com/apache/nuttx/pull/6447) bcm43xxx: Remove bcmf_txavail_work and resue bcmf_tx_poll_work
         ans init
+  uint32_t levl;
+  levl = classd_get_atten(vol, AUDIO_BALANCE_RIGHT - bal);
```

Comment:
could we avoid add so many ignore line?

Reply:
how about we leave this warning alone without adding many rule to the config file, and let the late contributor who is more familiar with the code to decide what's the best way to do. If we add these uncertain words/lines blindly, other people can't know some spell improvement may exist.